packages/eslint-plugin-qwik/package.json

```diff
@@ -28,7 +28,7 @@
   "license": "MIT",
   "main": "dist/index.js",
   "peerDependencies": {
-    "eslint": "^8.57.0"
+    "eslint": "^9.7.0"
```

Comment:
this will just force everyone to use v9 but it doesn't do anything to make us use v9